avm/Cargo.toml

```diff
@@ -5,7 +5,7 @@ license = "AGPL-3.0"
 homepage = "https://alan-lang.org"
 documentation = "https://docs.alan-lang.org"
 repository = "https://github.com/alantech/alan"
-version = "0.1.22"
+version = "0.1.48"
```

Comment:
Is this the Rust version? Or why did you change it?

Reply:
this is the Rust version

Reply:
Not according to: https://doc.rust-lang.org/cargo/reference/manifest.html#the-version-field

Reply:
It appears you *can't* specify the Rust compiler version in Cargo? https://github.com/emk/heroku-buildpack-rust/issues/11 That explains why we've had such difficulties with the submitted projects compiling on our laptops.

Reply:
Tracing through several years of github issues leads me to: https://github.com/rust-lang/rustup/pull/1172

We apparently need to add a `rust-toolchain` file that contains, as far as I can tell, only the version of rust that we want to use?